### ISSUE TITLE
Blockstack authentication not working

### ISSUE THREAD

**Issue by sohkai** [citrusbyte/universal-wallet#41]
<img width="589" alt="screen shot 2018-08-23 at 11 09 48 am" src="https://user-images.githubusercontent.com/4166642/44516828-5edbd200-a6c6-11e8-8c36-c683c9262bbb.png">

**Comment by AquiGorka** [comment]
@sohkai thanks for the feedback, I am aware `Blockstack` released a new version of their `js` client, maybe that's the reason for this to happen.
Will research and come back to you.

**Comment by AquiGorka** [comment]
@sohkai I did a fresh install and it worked for me, no need to update `blockstack.js` just yet.
Can you let me know how you arrived at such screen?

Are you using the live site (dirua.exchange)? testnet (testnet.dirua.exchange)? or locally?
If locally, did you follow these [steps](https://github.com/citrusbyte/universal-wallet/tree/master/packages/webapp)
OS, browser, blockstack via installed app or web?

Thanks

**Comment by sohkai** [comment]
Hey! Sorry, realized I was a bit light on the details.

I was trying this both with the Blockstack webapp as well as the macOS app. This happens on both the main and test net versions.

Browser was Chrome 68.0.3440.106.

**Comment by AquiGorka** [comment]
@sohkai,

TLDR;
- I am using [`surge`](http://surge.sh/) to host Dirua and decided to setup subdomains to use the different Major versions of Dirua (that's why you get redirected to v1.dirua.exchange) - because the data Blockstack saves is related to the domain/subdomain, so this way anyone using `v1` would not loose access to their data if `v2` came out and so forth.
- Not sure why but the `manifest.json` file for each subdomain was not being returned by `surge`, instead Dirua's `react-router` was responsible for the route and did not return the correct `json` file that Blockstack uses to show info when you grant access to log in (thus the error you pasted).
- Sometime ago, I decided to use a top domain `manifest.json` file to fix the issue. Apparently Blockstack now enforces the manifest file to come from the same subdomain and the issue re-appeared.

I moved the `manifest.json` file to a ´static´ folder within the app's `public` folder and it seems `surge` is returning the correct file now and I am able to sign in.

Let me know if the issue was fixed for you too.

**Comment by sohkai** [comment]
Interesting, I was wondering what that electron app was for :).
